### PR TITLE
Description of C and C++ installation

### PR DESCRIPTION
**Pull request by mruprich**
I tried to write a little something about running C and C++ under Fedora. Please tell me if you like it or tell me what should I change or add or remove.

### CODE REVIEW

**Comment by strzibny** [comment]
My feedback:

* I would split C and C++ content
* I would mention both `gcc` and `clang` installation

**Comment by phracek** [comment]
I would be also glad if you are able to describe a bit how automake and cmake works.
@kdudka Could you please review it?

File: tech/languages/c/about.md [hunk 1]
```diff
@@ -4,5 +4,67 @@ page: c
...
+### C
+
+C is a compiled language and as such it needs a compiler. To install a compiler, simply type:
+
+```
+# sudo dnf install gcc
+```
+
+This command should download and install package with compiler called gcc.
+To compile a program written in C, simply run the gcc like this:
```
Comment (kdudka) [inline tech/languages/c/about.md]: To compile and link... (compilation only mode can be enabled by the -c option of gcc)

File: tech/languages/c/about.md [hunk 2]
```diff
@@ -4,5 +4,67 @@ page: c
...
+# man gcc
+```
+
+### C++
+
+Like C, C++ is also a compiled language. It has many similarities with C and to some extent it is safe to say that C is
+subset of C++ and C++ supports every programming technique supported by C. Next command shows how to install C++ compiler:
+
+```
+# sudo dnf install g++
```
Comment (kdudka) [inline tech/languages/c/about.md]: ```# sudo dnf install gcc-c++```

File: tech/languages/c/about.md [hunk 3]
```diff
@@ -4,5 +4,67 @@ page: c
...
+```
+
+To compile your program you can do the same as was described above with C:
```
Comment (kdudka) [inline tech/languages/c/about.md]: To compile and link ....

**Comment by strzibny** [comment]
I would probably prefer to write \`file\` at the beginning of sentence rather than File (with capital F changing the name of the file or abbreviation). However I am unsure what's the right way to do it in English.

**Pull request merged**